src/Command/MarkMigrated.php

```diff
@@ -50,23 +51,47 @@ protected function configure()
             ->setDescription('Mark a migration as migrated')
             ->addArgument(
                 'version',
-                InputArgument::REQUIRED,
-                'What is the version of the migration? Use the special value `all` to mark all migrations as migrated.'
+                InputArgument::OPTIONAL,
+                'DEPRECATED: use `bin/cake migrations mark_migrated`'
```

Comment:
It should be ``DEPRECATED: use `bin/cake migrations mark_migrated --target=VERSION` ``